packages/parity-electron/src/runParity.ts

```diff
@@ -95,7 +95,7 @@ export const runParity = async (
     }
 
     // If the exit code is not 0, then we show some error message
-    if (Object.keys(parityArgv()).length > 0) {
+    if (parityArgv().length > 0) {
       app.exit(1);
     } else {
       onParityError(new Error(`Exit code ${exitCode}, with signal ${signal}.`));
```

Comment:
yeah, I think since the end result is the same in both cases, we might as well simplify :+1: 

> In fether, we're launching with --light, so parityArgv().length is always >0

not necessarily ; `--light` is part of `additionalFlags` ; `parityArgv()` are the fether options passed down to parity

Reply:
Ah yeah, true.

I'll merge this one as is, but as you proposed once, it'll be much cleaner to remove `cli` from `@parity/electron` and only take flags from `additionalFlags` in this function.